app/uploaders/listing_image_uploader.rb

```diff
@@ -1,7 +1,11 @@
 # encoding: utf-8
 class ListingImageUploader < CarrierWave::Uploader::Base
   include CarrierWave::RMagick
-  storage :file
+  if Rails.env.production?
```

Comment:
I merged and pulled your first request and it wouldn't let me rake will this fix it?